doc/news/_preparation_next_release.md

```diff
@@ -261,6 +261,7 @@ you up to date with the multi-language support provided by Elektra.
 - We now test the [Directory Value Plugin](https://www.libelektra.org/plugins/directoryvalue) with additional test data. _(René Schwaiger)_
 - <<TODO>>
 - The [CFramework](https://master.libelektra.org/tests/cframework) now also compares the names of meta keys. _(René Schwaiger)_
+- Java test_keySetCut_shouldPass and related documentation were fixed _(Miruna Orsa)_
```

Comment:
Please format this file with [Prettier](https://prettier.io). Otherwise the test [check_formatting](https://github.com/ElektraInitiative/libelektra/blob/master/tests/shell/check_formatting.sh) will fail. If you want, you can also follow the [instructions printed by the build server](https://build.libelektra.org/jenkins/blue/organizations/jenkins/libelektra/detail/PR-2557/1/pipeline/223) to fix the problem.

It might also make sense to surround the text “`test_keySetCut_shouldPass`” with back-ticks (`` ` ``) to mark it as code snippet. Otherwise some Markdown parser will assume you meant to [emphasize](https://daringfireball.net/projects/markdown/syntax#em) the term “keySetCut”.

Reply:
Thank you for the feedback, I will fix this today.